x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/api_key/40_role_descriptors.yml

```diff
@@ -0,0 +1,332 @@
+---
+setup:
+  - skip:
+      features: headers
```

Comment:
Adding a new YAML file instead of adding matches for role_descriptors to existing ones because:
* The matches are quite verbose and they clutter the existing tests, while keeping them separately is easier to have more targeted setup.
* It can be even more verbose once we add support for limited_role_descriptors
* Matching multiple API keys in response of QueryApiKey requires sorting and it is easier to setup in a new test (than retrofitting into existing ones)

Reply:
Changed to `40_view_role_descriptors.yml`.